frontend/web/components/FeatureHistory.tsx

```diff
@@ -56,57 +39,37 @@ const FeatureHistoryPage: FC<FeatureHistoryPageType> = ({ match, router }) => {
       is_live: true,
       page,
     },
-    { skip: !env || !feature },
+    { skip: !environmentId || !feature },
   )
   const [selected, setSelected] = useState<TFeatureVersion | null>(null)
   const live = data?.results?.[0]
   const [compareToLive, setCompareToLive] = useState(false)
   const [diff, setDiff] = useState<null | string>(null)
   const versionLimit = 3
   return (
-    <div className='container app-container'>
-      <PageTitle title={'History'}>
-        <div>
-          View and rollback history of feature values, multivariate values and
-          segment overrides.
-        </div>
-      </PageTitle>
-      <div className='row'>
-        <div className='col-md-4'>
-          <div className='flex-row'>
-            <label className='mb-0'>Feature</label>
-            <div className='flex-fill ml-2'>
-              <FlagSelect
-                placeholder='Select a Feature...'
-                projectId={match.params.projectId}
-                onChange={(flagId: string) => {
-                  router.history.replace(
-                    `${document.location.pathname}?feature=${flagId}`,
-                  )
-                }}
-                value={feature ? parseInt(feature) : null}
-              />
-            </div>
-          </div>
-        </div>
+    <div>
+      <h5>Change History</h5>
+      <div>
+        View and rollback history of feature values, multivariate values and
+        segment overrides.
       </div>
       <div className='mt-4'>
-        {!!versionLimit && (
-          <PlanBasedBanner
-            className='mb-4'
-            force
-            feature={'VERSIONING'}
-            theme={'page'}
-          />
-        )}
+        {/*{!!versionLimit && (*/}
```

Comment:
Is there any reason to keep this commented?

Reply:
Yes, because we're adjusting the version limit to be time based and I want to keep this here

Reply:
It'll be updated next week